### PR TITLE
Personal information UI index component added.

### PR DESCRIPTION
**Pull request by CanadianPhilipBarkow**
[ADO Work Item 2773](https://dev.azure.com/DTS-STN/Canada%20Dental%20Care%20Plan/_workitems/edit/2773)

### CODE REVIEW

**Comment by gregory-j-baker** [comment]
What's up with this? @CanadianPhilipBarkow do you need a hand fixing up your branch?

image

File: frontend/app/routes/_gcweb-app.personal-information._index.tsx [hunk 1]
```diff
@@ -25,28 +25,29 @@ export async function loader({ request }: LoaderFunctionArgs) {
 export default function PersonalInformationIndex() {
   const { user } = useLoaderData<typeof loader>();
   const { t } = useTranslation(i18nNamespaces);
+  const languageName = user?.preferredLanguage === t('personal-information:index.french-abbreviation') ? t('personal-information:index.french-language-name') : t('personal-information:index.english-language-name');
```
[the remark this answers is not in the document]
Comment (CanadianPhilipBarkow) [reply frontend/app/routes/_gcweb-app.personal-information._index.tsx]: Makes sense, it has now been implemented.

File: frontend/app/routes/_gcweb-app.personal-information._index.tsx [hunk 2]
```diff
@@ -25,28 +25,29 @@ export async function loader({ request }: LoaderFunctionArgs) {
...
-        <dd>{user?.lastName}</dd>
-        <dt>{t('personal-information:index.phone-number')}</dt>
-        <dd>{user?.phoneNumber}</dd>
-        <dt>{t('personal-information:index.home-address')}</dt>
-        <dd>{user?.homeAddress}</dd>
-        <dt>{t('personal-information:index.mailing-address')}</dt>
-        <dd>{user?.mailingAddress}</dd>
-        <dt>
+      <section>
+        <h3>{t('personal-information:index.first-name')}</h3>
```
[the remark this answers is not in the document]
Comment (CanadianPhilipBarkow) [reply frontend/app/routes/_gcweb-app.personal-information._index.tsx]: That's needed for accessibility purposes, right?

Now implemented.

[the remark this answers is not in the document]
Comment (nicholas-ly) [reply frontend/app/routes/_gcweb-app.personal-information._index.tsx]: Yep - resolving.

**Review by nicholas-ly — APPROVED** [review]
Looks good. I will approve so this can get merged. 

Here are a few nit pick suggestions for a future PR:
- Rather than defining two `const` for `VALUES` and `LanguageEnum`, we can specify the `enum(..)` all in one line: `preferredLanguage: z.enum(['fr', 'en']).optional()`
- We currently have two panels for home address and mailing address but the "Change" button redirects the user to the same page. Maybe we should combine these two panels into one "Addresses" panel
- I don't feel it's necessary to have the "First name" and "Last name" show on this page, or if they should show, it should be in one panel
- We're using an envelope icon everywhere, even for phone number. Consider changing the icons to better match content of the panel

**Pull request merged**